src/IIIFPresentation/Migrator/Dockerfile

```diff
@@ -0,0 +1,18 @@
+﻿FROM mcr.microsoft.com/dotnet/runtime:8.0 AS base
```

Comment:
How does this and `Dockerfile.Migrator` compare? Are both required? If this is the one auto-added by Rider, is it best to just use this for the 'real' dockerfile OR deleting this in favour of docker file at root? rather than having multiple?